docs/ii-spec.mdx

```diff
@@ -172,6 +173,8 @@ This section describes the Internet Identity Service from the point of view of a
 
     -   the `derivationOrigin`, if present, indicates an origin that should be used for principal derivation instead of the client origin. Internet Identity will only accept values that are also listed in the HTTP resource `/.well-known/ii-alternative-origins` of the corresponding canister (see [Alternative Frontend Origins](#alternative-frontend-origins)).
 
+    -   the `autoSelectMatchingIdentity`, if present, indicates the textual representation of this dapp's principal for which the delegation is requested. If it is known to Internet Identity, it will skip the identity selection and immediately prompt for authentication. This feature can be used to streamline re-authentication after a session expiry.
```

Comment:
Do you have a list of changes downstream for this?

First thoughts are auth-client and [II docs](https://internetcomputer.org/docs/current/references/ii-spec/#client-authentication-protocol)

Anything else?

Reply:
As discussed, we only need to have the docs rebuilt after merging.